docs/reference/search/search-your-data/semantic-search-inference.asciidoc

```diff
@@ -9,16 +9,20 @@ The instructions in this tutorial shows you how to use the {infer} API workflow
 
 IMPORTANT: For the easiest way to perform semantic search in the {stack}, refer to the <<semantic-search-semantic-text, `semantic_text`>> end-to-end tutorial.
 
-The following examples use Cohere's `embed-english-v3.0` model, the `all-mpnet-base-v2` model from HuggingFace, and OpenAI's `text-embedding-ada-002` second generation embedding model.
+The following examples use the:
+
+* `embed-english-v3.0` model for https://docs.cohere.com/docs/cohere-embed[Cohere]
+* `all-mpnet-base-v2` model from https://huggingface.co/sentence-transformers/all-mpnet-base-v2[HuggingFace]
+* `text-embedding-ada-002` second generation embedding model for OpenAI
+* models available through https://ai.azure.com/explore/models?selectedTask=embeddings[Azure AI Studio] or https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models[Azure OpenAI]
+* `text-embedding-004` model for https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/text-embeddings-api#generative-ai-get-text-embedding-python_vertex_ai_sdk[Google Vertex AI]
```

Comment:
Should we link to https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/text-embeddings-api instead of the python vertex ai specific section? The top of the page has a model overview